crud/get.lua

```diff
@@ -14,8 +14,65 @@ local get = {}
 
 local GET_FUNC_NAME = '_crud.get_on_storage'
 
-local function get_on_storage(space_name, key)
-    dev_checks('string', '?')
+local function get_field_metadata(full_metadata, field)
+    dev_checks('table', 'string')
+
+    for _, tuple in ipairs(full_metadata) do
+        if tuple['name'] == field then
+            return tuple
+        end
+    end
```

Comment:
This code looks a bit suboptimal. I suggest to cache field map.

Small snippet:

```lua
local cache = setmetabale({}, {__mode = 'k'}) -- https://www.lua.org/pil/17.html

...
local metadata_cache = cache[full_metadata]
if metadata_cache ~= then
    return metadata_cache[field]
end

cache[full_metadata] = {}
for i, field in ipairs(full_metadata) do 
    cache[full_metadata][field.name] = field
end

...
return cache[full_metadata][field]
```